src/client/responses/DisplayContainer.tsx

```diff
@@ -35,10 +40,13 @@ export const DisplayContainer = (props: Props) => {
     const [data, setData] = React.useState<{
         MC: Array<[string, string]>;
         SA: Array<[string, string]>;
-    }>({ MC: [], SA: [] });
+        FF: Array<[string, string]>;
+    }>({ MC: [], SA: [], FF: [] });
 
     const updateForms = React.useCallback(() => {
         getForms({ sid: props.sessionID });
+        const notification = new Audio("/public/notification.wav");
```

Comment:
These should be stored in refs otherwise new instances need to be made per effect